compliance/collection/nodescanv2/fake_nodescan.go

```diff
@@ -0,0 +1,83 @@
+package nodescanv2
+
+import (
+	timestamp "github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/logging"
+	scannerV1 "github.com/stackrox/scanner/generated/scanner/api/v1"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+// FakeNodeScanner can be used to send fake messages that would be emitted by NodeScanV2
+type FakeNodeScanner struct {
+}
+
+// Scan returns a fake message in the same format a real NodeScanV2 would produce
+func (f *FakeNodeScanner) Scan(nodeName string) (*storage.NodeScanV2, error) {
+	log.Infof("Generating fake scan result message...")
+	msg := &storage.NodeScanV2{
+		NodeId:   "",
+		NodeName: nodeName,
+		ScanTime: timestamp.TimestampNow(),
+		Components: &scannerV1.Components{
+			Namespace: "Testme OS",
+			OsComponents: []*scannerV1.OSComponent{
+				{
+					Name:      "Test Component",
+					Namespace: "tos4",
+					Version:   "42.24",
+					AddedBy:   "FakeLayer",
+				},
+				{
+					Name:      "sqlite-libs",
+					Namespace: "centos:8",
+					Version:   "3.26.0-6.el8",
+					AddedBy:   "FakeLayer",
+				},
+			},
+			RhelComponents: []*scannerV1.RHELComponent{
+				{
+					Name:      "vim-minimal",
+					Namespace: "rhel:7",
+					Version:   "2:7.4.629-6.el7.x86_64",
+					Arch:      "x86_64",
+					Module:    "FakeMod",
+					Cpes:      []string{"cpe:/a:redhat:enterprise_linux:7::baseos"},
+					AddedBy:   "FakeLayer",
+					Executables: []*scannerV1.Executable{
```

Comment:
let's ignore executables as well since this is for Active Vuln mgmt, and I don't think we plan on having that in the first iteration